public/api-enterprise-spec.json

```diff
@@ -685,216 +685,6 @@
         }
       }
     },
-    "/datasources/{datasourceId}/disable-permissions": {
```

Comment:
these changes come from another repository: [DatasourceACL: Remove deprecated datasource permissions endpoints](https://github.com/grafana/grafana-enterprise/pull/5880)
we need to introduce a similar [swagger-gen](https://drone.grafana.net/grafana/grafana/146375/7/3) step there too

Reply:
Yes, I was a bit confused by this. But I looked at the other repo and these endpoints were removed.